src/main/kotlin/no/nav/tiltaksarrangor/repositories/model/DeltakerlisteDbo.kt

```diff
@@ -9,9 +9,21 @@ data class DeltakerlisteDbo(
 	val navn: String,
 	val status: DeltakerlisteStatus,
 	val arrangorId: UUID,
-	val tiltakNavn: String,
+	private val _tiltakNavn: String,
 	val tiltakType: String,
 	val startDato: LocalDate?,
 	val sluttDato: LocalDate?,
 	val erKurs: Boolean
-)
+) {
+
+	val tiltakNavn = _tiltakNavn
```

Comment:
Kanskje man kunne latt feltet være som det var, og heller ha en egen metode for å hente visningsnavn for tiltaket?

Reply:
Ja, kan godt det, faren er jo at det da blir mulig å bruke "feil" navn ellers i appen. Men kan godt gjøre det sånn!